src/data/bedrock/block/convert/BlockStateToObjectDeserializer.php

```
@@ -887,6 +887,7 @@
 		$this->mapSimple(Ids::ANCIENT_DEBRIS, fn() => Blocks::ANCIENT_DEBRIS());
 		$this->mapSimple(Ids::ANDESITE, fn() => Blocks::ANDESITE());
 		$this->mapSimple(Ids::BARRIER, fn() => Blocks::BARRIER());
+		$this->mapSimple(Ids::STRUCTURE_VOID, fn() => Blocks::STRUCTURE_VOID());
```

PHPStan (2.1.15)

Call to an undefined static method pocketmine\block\VanillaBlocks::STRUCTURE_VOID().


(staticMethod.notFound)